<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lightDialog</class>
 <widget class="QDialog" name="lightDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Intensity</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="intensityBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="posZBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="posYBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="posXBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="colRBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="colGBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="colBBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lightDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lightDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
